components/common/TVFocusableItem.tsx

```diff
@@ -0,0 +1,59 @@
+import type React from "react";
+import { PropsWithChildren } from "react";
+import { Platform, Pressable, TouchableOpacity, View } from "react-native";
+import Animated from "react-native-reanimated";
+import { useTVFocusAnimation } from "@/hooks/useTVFocusAnimation";
+
+interface TVFocusableItemProps extends PropsWithChildren {
+  onPress?: () => void;
+  className?: string;
+  style?: any;
+  disabled?: boolean;
+}
+
+/**
+ * Reusable component for TV-focusable items with elevation effects
+ * Automatically handles platform detection and provides consistent animations
+ */
+export const TVFocusableItem: React.FC<TVFocusableItemProps> = ({
```

Comment:
If this is a platform independent component then we could just name it FocusableItem?

Reply:
Makes sense